content/vendor/guides/quickstart.md

```diff
@@ -48,20 +48,17 @@ For now, click on the Releases item on the left menu and then click the "Create
 
 You should now see a YAML editor where you can define how you application will work and the integration with KOTS functionality.
```

Comment:
You should now see a YAML editor where you can define how **your** application...

Reply:
```suggestion
You should now see a YAML editor where you can define how your application will work and the integration with KOTS functionality.
```